### PR TITLE
More cleaning up

### PR DESCRIPTION
**Pull request by jimbobbennett**
Removing unnecessary documents - check each commit for details

### CODE REVIEW

File: Educator Resources/Azure Guides/MobileDevelopment.md [hunk 1]
```diff
@@ -1,150 +0,0 @@
-![](media/image1.png)
-
-# Mobile Development
```
Comment (shanamatthews) [inline Educator Resources/Azure Guides/MobileDevelopment.md:3]: Any way we can update this rather than delete?

Comment (jimbobbennett) [reply Educator Resources/Azure Guides/MobileDevelopment.md:3]: Not really - the Azure Mobile services are going away and we don't have a decent replacement...

**Comment by jimbobbennett** [comment]
> I think it would be nicer if we could update ALL of these guides (including mobile development) rather than delete them. Otherwise, all these changes look great!

I've only deleted the docx versions, left the markdown versions in place, except for the mobile one where we don't have a good service to update them to.

File: Educator Resources/Azure Guides/MobileDevelopment.md (hunk 1 above)
Comment (shanamatthews) [reply Educator Resources/Azure Guides/MobileDevelopment.md:3]: Gotcha. Ok, sounds good!

**Pull request merged**